core/client/fs/src/test/java/alluxio/client/file/BaseFileSystemTest.java

```diff
@@ -452,37 +490,114 @@ public void unmountException() throws Exception {
   }
 
   /**
-   * Ensures Warnings are logged or exceptions are thrown when an {@link AlluxioURI} with a
-   * scheme://host is passed.
+   * Ensures warnings are logged and exception is thrown when an {@link AlluxioURI} with an invalid
```

Comment:
this should only have one warning, right?

Reply:
Well because the authority and scheme are separate fields and can be passed independently I thought logging both warnings would be better. A user could pass a URI `alluxio:///...`, `alluxio://host:port/...`, `//host:port/...` or `/...`.  Whether a scheme or authority is passed is independent. It could have one, both, or none. So I figured logging a separate warning for each would make sense.

Reply:
ah right, agree that the warnings should appear independently. On further consideration, I think it might be a little too harsh to warn on using the correct `alluxio://` scheme, maybe we could put it at debug-level or info-level instead? `warn` is usually for things where we suspect they might cause problems, but `alluxio://` in the URI, though unnecessary, is unlikely to cause any harm. Supplying an authority deserves a `warn` since the user will likely expect that we are using their supplied `host/port`, when in fact we aren't.